tests/test_urlparse.py

```diff
@@ -1137,7 +1156,7 @@ def test_unwrap(self):
         url = urlparse4._unwrap('<URL:type://host/path>')
         self.assertEqual(url, 'type://host/path')
 
-
+@pytest.mark.skip(reason="we dont need this test")
```

Comment:
Same here - if we're not going to need this in the future, it's fine to remove it completely